kinetic/distribution.yaml

```diff
@@ -4160,6 +4160,22 @@ repositories:
       url: https://github.com/corb555/iot_bridge.git
       version: kinetic-devel
     status: developed
+  ipcamera_driver:
+    doc:
+      type: git
+      url: https://github.com/alireza-hosseini/ipcamera_driver.git
+      version: master
+    release:
+      tags:
+        release: release/kinetic/{package}/{version}
+      url: https://github.com/alireza-hosseini/ipcamera_driver-release
+      version: 0.1.1-1
+    source:
+      test_pull_requests: true
```

Comment:
Just checking: Have you followed the [instructions](http://wiki.ros.org/buildfarm/Pull%20request%20testing) to provide access to this repo or the org unit to the @ros-pull-request-builder user? If not, please either remove this line or provide the necessary access as described in the wiki page.

Reply:
@mikaelarguedas Yes, I did a few minutes ago.

Reply:
Awesome thanks for confirming :+1: